pyscriptjs/src/python/pyscript.py

```diff
@@ -124,6 +123,21 @@ def write(element_id, value, append=False, exec_id=0):
         )
 
 
+CURRENT_PY_SCRIPT_TAG = None
+
+
+def set_current_display_target(element):
+    global CURRENT_PY_SCRIPT_TAG
+    CURRENT_PY_SCRIPT_TAG = element
+
```

Comment:
nitpick, but since [naming is one of the two hard things in computer science](https://martinfowler.com/bliki/TwoHardThings.html).
When I first read this code, I though that `element` was supposed to be an instance of the `Element` class, but then I realized that it's actually and element *id*.
So, I would just call it `target_id`.

Reply:
+1

Reply:
This code is not valid anymore :)